content/30-use-pah/_index.md

```diff
@@ -7,106 +7,131 @@ weight = 30
+* After the sync process has finished, check the certified collections are visible on PAH.
 
-#### Sync selected community collections from Ansible Galaxy
+#### Integrate your PAH in Automation Controller
 
-Galaxy is configured as the remote `community` out of the box. Follow the instructions to configure the synchronization.
+Automation Controller doesn't know about your Private Automation Hub, yet. The first step is to create a new Credential pointing to your PAH:
 
-* Create a regular requirements.yml file pointing to the collection you want, we'll use the one from the docs:
+Get access URL and Token fromm your PAH and put them down somewhere:
 
-```yaml
-collections:
-  # Install a collection from Ansible Galaxy.
-  - name: containers.podman
-```
+* In PAH, **Collections**->**Repository management**
+* Get the URLs for the **local** repos **community**, **published** and **rh-certified**
+* Get the access token from **Collections**->**API Token**
 
-* Go to Repo Management, click the **Remote** tab again
+Then configure three new credentials in Automation Controller for the three URLs:
 
-* Edit the `community` remote
+* **Name**: PAH {Community,Published,RH-Certified}
+* **Organization**: default
+* **Credential Type**: Ansible Galaxy/Automation Hub API Token
+* **Galaxy Server URL**: <the respective community repo URL>
+* **API Token**: <the token>
 
-* In **YAML requirements** upload the  `requirements.yml` file from your local machine.
+Now the access information is configured but not used by Automation Controller. This is done on **Organization** level.
 
-* Click **Save**
+* Open the `default` **Organization** in Automation Controller and click **Edit**
+* Have a look at the **Galaxy Credentials** field. There is only one credential pointing to Ansible Galaxy. This is the default.
+* Remove the credential and add the one pointing to your PAH.
 
-* In the **Remote** overview tab click **Sync** for the `community` remote
-
-Verify the sync of the collections in **Collections** -> **Collections**, switch the repository filter with the dropdown at the top. There should be a lot of content in the `Red Hat Certified` repo and one collection in the `Community` repo. The 'published' filter will not find anything, since we haven't uploaded any collections we created ourselves.
+{{% notice note %}}
+You could add more credentials here, the order of these credentials sets precedence for the sync and lookup of the content.
+{{% /notice %}}
 
 ### Test Private Automation Hub Integration
 
-Now check that your Automation Controller can actually use the content from your PAH:
+Now check that Automation Controller can actually use the content from your PAH by creating a **Job Template** that requires a collection that is not part of the included Execution Environments:
 
 * Create a new **Project** pointing here: `https://github.com/ansible-learnfest/ee-flow.git`
+  * Have a look at the content on GitHub, esp the `collections/requirements.yml` file
```

Comment:
in my environment the project sync fails - Controller is unable to fetch collection from PAH (connection timeout)